### PR TITLE
Dls 4283 Library upgrades Play 2.6

### PR DESCRIPTION
**Pull request by gavinharris43**

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.9","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/gavinharris/Library/Application Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```
Comment (calinursaciuc) [inline .bsp/sbt.json:1]: Remove this from commit and add .bsp into gitignore

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
+resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
+resolvers += Resolver.jcenterRepo
 
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
+addSbtPlugin("com.github.sbt" % "sbt-release" % "1.0.15")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.23")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.6")
```
Comment (calinursaciuc) [inline project/plugins.sbt:9]: use sbt-plugin 2.7.9 if possible so we have latest 2.7 version

File: .bsp/sbt.json (hunk 1 above)
Comment (gavinharris43) [reply .bsp/sbt.json]: done

File: project/plugins.sbt (hunk 2 above)
Comment (gavinharris43) [reply project/plugins.sbt]: done

**Review by calinursaciuc — APPROVED** [review]

**Pull request merged**